internal/cmd/auth/auth_login.go

```diff
@@ -159,6 +161,15 @@ func completeLogin(ctx context.Context, opts *loginOptions) error {
 				return err
 			}
 		}
+	} else if axiom.IsAPIToken(opts.Token) && deploymentKind == typeCloud && opts.OrganizationID == "" {
+		if err := survey.AskOne(&survey.Input{
```

Comment:
API tokens don't need an org id

Reply:
Thanks for the heads up. Yeah, it shouldn't. Made that change to make the error disappear. But the `ErrMissingOrgID` is actually valid. Something broken with `axiom-go`: https://github.com/axiomhq/axiom-go/blob/main/axiom/client.go#L184-L198.